### DISCUSSION TITLE
プロジェクトの外部紹介について

### DISCUSSION THREAD

**Discussion by miurahr** [OSGeo-jp/OSGeoLive-doc-omegat#35] [Show and tell]
このとりくみに関連して、その成果や利用している技術について、日本のLibreOfficeコミュニティに紹介したところ、とても興味をもってくれましたので、共有します。

https://youtu.be/jy4wsM4U8Wo

**Comment by sanak** [comment]
@miurahr 上記情報、ありがとうございます！ 🙇
